veneer/percona-server-mongodb-operator.yaml

```diff
@@ -4,4 +4,4 @@ GenerateMinorChannels: false
 Stable:
   Bundles:
   - Image: docker.io/percona/percona-server-mongodb-operator:1.18.0-community-bundle
-  - Image: docker.io/percona/percona-server-mongodb-operator:1.19.0-community-bundle
+  - Image: docker.io/perconalab/percona-server-mongodb-operator:1.19.1-community-bundle
```

Comment:
leaving a comment to remember to change `perconalab` to `percona` before merging

Reply:
Done